jsconfig.json

```diff
@@ -3,13 +3,12 @@
   "compilerOptions": {
     "baseUrl": "./app",
     "paths": {
-      "api/*": [ "api/*" ],
-      "assets/*": [ "assets/*" ],
-      "behaviors/*": [ "modules/general/behaviors/*" ],
-      "lib/*": [ "lib/*" ],
-      "modules/*": [ "modules/*" ],
+      "#assets/*": [ "assets/*" ],
+      "#behaviors/*": [ "modules/general/behaviors/*" ],
```

Comment:
```suggestion
      "#behaviors/*": [ "modules/general/behaviors/*"
      "#components/*": [ "modules/general/components/*" ],
```

Reply:
with 2fcbf9bd4 jsconfig.json is a derived from package.json